### PR TITLE
update example notebook

### PR DESCRIPTION
**Pull request by rhagarty**
- needed to match updated version of the notebook

### CODE REVIEW

File: examples/facial-recognition.ipynb [hunk 1]
```diff
@@ -173,7 +177,7 @@
    "outputs": [],
    "source": [
     "# Fill here the bucket name you created in COS Dashboard \n",
-    "BUCKET = 'pywren-bucket-for-*****'"
+    "BUCKET = 'pywren-bucket-for-richh-2'"
```
Comment (scottdangelo) [inline examples/facial-recognition.ipynb]: Did you mean to change this? I like things like '******' to make it clear that it is a placeholder, also easy to find.

File: examples/facial-recognition.ipynb [hunk 2]
```diff
@@ -205,10 +209,10 @@
    "outputs": [],
    "source": [
     "cos_credentials = {\n",
-    "  \"apikey\": \"*****\",\n",
+    "  \"apikey\": \"br-NqZ-069lTH84qsGgRcmZ14Z8FirqEdvlNvK_lugzs\",\n",
```
Comment (scottdangelo) [inline examples/facial-recognition.ipynb]: Definitely don't want this!!!

File: examples/facial-recognition.ipynb [hunk 3]
```diff
@@ -347,7 +351,7 @@
     "config = {\n",
     "          'ibm_cf':  {'endpoint': 'https://openwhisk.ng.bluemix.net', \n",
     "                      'namespace': 'IBM Cloud Storage_dev', \n",
-    "                      'api_key': '*****'}}\n",
+    "                      'api_key': '7a4d1a77-2429-43b1-b6ed-a2b438e15bea:RVVEdpPFLAuuTwFXjjKujPKY0hUOEzt6nQ6O7NwyonHeF7OdAm77Uc34GL2wQHDx'}}\n",
```
Comment (scottdangelo) [inline examples/facial-recognition.ipynb]: this goes too

File: examples/facial-recognition.ipynb [hunk 4]
```diff
@@ -981,15 +1012,15 @@
    "source": [
     "# @hidden_cell\n",
     "wml_credentials = {\n",
-    "  \"apikey\": \"*****\",\n",
+    "  \"apikey\": \"vfU093NmXmAmqBCKYoo8nYvLaTvSF7xveUZT6t4qqUyp\",\n",
```
Comment (scottdangelo) [inline examples/facial-recognition.ipynb]: bad

File: examples/facial-recognition.ipynb [hunk 5]
```diff
@@ -981,15 +1012,15 @@
...
     "  \"iam_apikey_description\": \"Auto generated apikey during resource-key operation for Instance - crn:v1:bluemix:public:pm-20:us-south:a/bc1bd51c396536dc7d5f81d5a4e19533:a8ab45b5-055a-40d9-8418-0b26bae17433::\",\n",
     "  \"iam_apikey_name\": \"auto-generated-apikey-8d0c1b72-1bf6-40b9-824d-06ea3a67f148\",\n",
     "  \"iam_role_crn\": \"crn:v1:bluemix:public:iam::::serviceRole:Writer\",\n",
     "  \"iam_serviceid_crn\": \"crn:v1:bluemix:public:iam-identity::a/bc1bd51c396536dc7d5f81d5a4e19533::serviceid:ServiceId-22ebfeae-7e29-45ff-8c5f-097f6537858e\",\n",
     "  \"instance_id\": \"a8ab45b5-055a-40d9-8418-0b26bae17433\",\n",
-    "  \"password\": \"*****\",\n",
+    "  \"password\": \"c2b345af-372c-4a92-ad0f-e5b55ec82a4d\",\n",
```
Comment (scottdangelo) [inline examples/facial-recognition.ipynb]: bad

File: examples/facial-recognition.ipynb [hunk 6]
```diff
@@ -981,15 +1012,15 @@
...
     "  \"url\": \"https://us-south.ml.cloud.ibm.com\",\n",
-    "  \"username\": \"*****\"\n",
+    "  \"username\": \"8d0c1b72-1bf6-40b9-824d-06ea3a67f148\"\n",
```
Comment (scottdangelo) [inline examples/facial-recognition.ipynb]: not good

File: examples/facial-recognition.ipynb (hunk 1 above)
Comment (rhagarty) [reply examples/facial-recognition.ipynb]: good catch - fixed

**Pull request merged**